qutip/tests/test_wigner.py

```diff
@@ -36,11 +36,158 @@
 from numpy.random import rand
 from numpy.testing import assert_, run_module_suite, assert_equal
 
-from qutip.states import coherent, fock
-from qutip.wigner import wigner
+from qutip.states import coherent, fock, ket, bell_state
+from qutip.wigner import wigner, wigner_transformation, _parity
 from qutip.random_objects import rand_dm, rand_ket
 
 
+def test_wigner_bell1_su2parity():
+    psi = bell_state('00')
```

Comment:
Under the function definition there should be a 1 line docstring to describe the test.
Can be simple like `"testing bell su2 parity"`